projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/google-sheets-card.js

```diff
@@ -81,7 +81,7 @@ const GoogleSheetsCard = ( { isExpanded, onToggle, data, refreshStatus } ) => {
 				</div>
 			) : (
 				<div>
-					<p>
+					<p className="integration-card__description">
```

Comment:
This could be a component something like `<IntergrationCard.Description>` since the same html is being used across many different cards.